test/ngTouch/directive/ngClickSpec.js

```diff
@@ -49,6 +49,16 @@ describe('ngClick (touch)', function() {
   }));
 
 
+  it('should unwrap a jQuery-wrapped event object', inject(function($rootScope, $compile) {
+    element = $compile('<div ng-click="event = $event"></div>')($rootScope);
+    $rootScope.$digest();
+
+    browserTrigger(element, 'touchstart');
```

Comment:
You should check if `$rootScope.event` is correct after this line as well; otherwise the `touchstart` part of the patch is not tested.